src/components/WwwFrame/TheTipMessage.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+	<transition name="fade">
+		<div v-if="tipVisible" class="message-text text-center small-12" :class="typeClass">
+			<span class="message-content">
+				<div class="icon-wrapper">
+					<kv-icon :name="iconName" />
+				</div>
+				<p class="message">{{ tipMsg }}</p>
+			</span>
+			<a @click="closeTip" class="close-tip-message" aria-label="Close">
+				<kv-icon name="x" />
+			</a>
+		</div>
+	</transition>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import tipMessageData from '@/graphql/query/tipMessageData.graphql';
+import updateTipMessage from '@/graphql/mutation/updateTipMessage.graphql';
+import KvIcon from '@/components/Kv/KvIcon';
+
+export default {
+	components: {
+		KvIcon
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			tipVisible: false,
+			tipPersist: false,
+			tipInitUrl: '',
+			tipMsg: '',
+			tipMsgType: ''
+		};
+	},
+	computed: {
+		typeClass() {
+			// Valid options are 'warning' + 'error' otherwise use confirmation as default
+			if (this.tipMsgType === 'warning' || this.tipMsgType === 'error') {
+				return `message-text-${this.tipMsgType}`;
+			}
+			return 'message-text-confirmation';
+		},
+		iconName() {
+			// warning icon-info, error icon-error
+			if (this.tipMsgType === 'warning') {
+				return 'info';
+			} else if (this.tipMsgType === 'error') {
+				return 'error';
+			}
+			// default icon-confirmation
+			return 'confirmation';
+		}
+	},
+	methods: {
+		/*
+			Close tip message
+			- tipMsg + tipMsgType are retained when closed by clicking 'X'
+		*/
+		closeTip() {
+			this.apollo.mutate({
+				mutation: updateTipMessage,
+				variables: {
+					tipMsg: this.tipMsg,
+					tipMsgType: this.tipMsgType,
+					tipVisible: false,
+					tipPersist: false,
+					tipInitUrl: ''
+				}
+			});
+		}
+	},
+	created() {
+		// attach to and watch for updates to the query
+		this.apollo.watchQuery({ query: tipMessageData }).subscribe({
+			next: ({ data }) => {
+				// set/update our local data
+				this.tipMsg = _get(data, 'tipMsg');
+				this.tipMsgType = _get(data, 'tipMsgType');
+				this.tipVisible = _get(data, 'tipVisible');
+				this.tipPersist = _get(data, 'tipPersist');
+				this.tipInitUrl = _get(data, 'tipInitUrl');
+			}
+		});
```

Comment:
This could also be done with the `apollo` config option:
```
apollo: {
    query: tipMessageData,
    result({ data }) {
        this.tipMsg = _get(data, 'tipMsg');
        ...
    },
}
```